directory/src/main/java/org/ecclesia/directory/domain/Person.java

```diff
@@ -1,15 +1,18 @@
 package org.ecclesia.directory.domain;
 
+import java.io.Serializable;
 import javax.persistence.*;
-import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
 
 /**
  * Represents one person. Data transfer object.
  */
 @Entity
 @Table(name = "person")
-@Data
-public class Person {
+@Getter @Setter @NoArgsConstructor
```

Comment:
**Reporter**: Checkstyle
**Rule**: AnnotationLocationMostCases
**Severity**: WARN
**File**: /var/jenkins_home/workspace/ecclesia_PR-6/directory/src/main/java/org/ecclesia/directory/domain/Person.java L14

Annotation &#39;Setter&#39; should be alone on line.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1304528268>*